packages/components/modules/navigations/web/NavigationLayout/index.tsx

```diff
@@ -24,6 +24,7 @@ const NavigationLayout: FC<NavigationLayoutProps> = ({
   AccountMenuProps,
   ToolbarProps,
   children,
+  BodyProps,
```

Comment:
I think we could make this a bit more flexible and add `MainContainerProps` instead of a specific prop and also allow the main container to be overridden, something like:

```
import DefaultMainContainer from './MainContainer'

const NavigationLayout: FC<NavigationLayoutProps> = ({
  ...
  MainContainer = DefaultMainContainer,
  MainContainerProps = {},
}) => {
  ...
  <MainContainer isNavCentered {...MainContainerProps}>
          {children}
  </MainContainer>
```